static/coffee/flows/controllers.coffee

```diff
@@ -1873,6 +1898,24 @@ NodeEditorController = ($rootScope, $scope, $modalInstance, $timeout, $log, Flow
     if $scope.action.webhook_headers.length == 0
       $scope.addNewActionWebhookHeader()
 
+  $scope.addNewQuickReply = ->
+    $scope.showQuickReplyButton = false
+    if $scope.quickReplies.length < 11
+      addQuickReply = {}
+      addQuickReply[$scope.base_language] = ''
+      $scope.quickReplies.push(addQuickReply)
+
+  $scope.removeQuickReply = (a, index) ->
+    a.splice(index, 1)
+
+    if Object.keys($scope.action.quick_replies).length == 0
+      $scope.quickReplies.splice(index, 1)
```

Comment:
do we even need `a` since you're also modifying `$scope.quickReplies` directly? Aren't they the same thing?